runtime/browser-demo/browser-demo.js

```diff
@@ -15,6 +15,7 @@ let BrowserLoader = require("../browser-loader.js");
 let Resolver = require('../resolver.js');
 let SlotManager = require('../slot-manager.js');
 let Suggestinator = require("../suggestinator.js");
+let SuggestionManager = require('../suggestion.js').SuggestionManager;
```

Comment:
I borrowed this idea and renamed SuggestionManager to SuggestionComposer (I can handle the rename of SlotManager in the upcoming change of renaming its methods, if that's fine with you).

I also moved it to suggestion-composer.js file. Although I did keep the Suggestion class existence, and kept it in the same file, but this class is not exported now, so maybe it is alright?